spec/test-data/translations/glotpress-download/expected/values/available_languages.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--Warning: Auto-generated file, do not edit.-->
+<resources>
+  <string-array name="available_languages" translatable="false">
+    <item>en_US</item>
+    <item>es</item>
+    <item>pt_BR</item>
+    <item>it</item>
+    <item>zh_CN</item>
+    <item>zh_TW</item>
```

Comment:
For whatever reason, Android use two different standards to specify locale codes, in this file it uses the `zh_CN` notation while in the name of `res/values-*/` directories ut uses `zh-rCN`… 🤷  So at least we test that too here.